source/pages/index.jsx

```diff
@@ -17,20 +18,37 @@ export default ({ locals }) => (
       <Rhythm size="large">
         <Heading level="1">Hello World</Heading>
 
-        <Rhythm>
-          <Heading level="2">Pages</Heading>
-          <FileIndex items={getPagesIndexData(locals)} className="rhythm--small" />
-        </Rhythm>
+        {
+          ! isEmpty(getPagesIndexData(locals))
+          ?
+          <Rhythm>
+            <Heading level="2">Pages</Heading>
+            <FileIndex items={getPagesIndexData(locals)} className="rhythm--small" />
+          </Rhythm>
+          :
+          null
+        }
 
```

Comment:
the isEmpty dependency can be replaced with array.length since the `get*IndexData` functions will always return an array – nothing fancy :man_in_tuxedo:  then the code style i have been using for ternary operators in expressions is as below.

```js
{ !pagesIndexData.length
  ? <Rhythm>
      <Heading level="2">Pages</Heading>
      <FileIndex items={pagesIndexData} className="rhythm--small" />
    </Rhythm> 
  : null }
```